toDoList.json

```diff
@@ -0,0 +1 @@
+{"tasks":[{"description":"Task one","status":false},{"description":"Task two","status":true},{"description":"Task three","status":true},{"description":"four","status":false},{"description":"five tasks","status":false}]}
```

Comment:
This file should not be committed to the repo (as [per your `.gitignore`](https://github.com/rosadiaz/homeworks/pull/5/files#diff-a084b794bc0759e7a6b77810e01874f2R199))